tests/unit/s2n_signature_scheme_test.c

```diff
@@ -13,14 +13,16 @@
  * permissions and limitations under the License.
  */
 
-#include "tls/s2n_signature_scheme.c"
+#include "tls/s2n_signature_scheme.h"
```

Comment:
This typo made me think I was going insane.

My pointer comparisons in this test weren't working in the OSX github action, and ONLY in the OSX github action.  I couldn't repro locally. Like `sig_scheme == all_prefs->signature_schemes[all_i]` was false for the same signature scheme. I added print statements to print the pointer addresses. They were different?!

It was because of this typo. s2n_signature_preferences_all was the duplicate copy from this include, using its own local copies of the signature schemes. The signature schemes from security_policy_selection were the originals used by the rest of the library -_-